public/lib/latencyBasedRouting.js

```diff
@@ -35,7 +35,8 @@
         this.latencyHttpTestRequest = [];
         this.numServersResponded = 0;
         this.trackingServerInfo = [];
-        this.latencyTimeout = timeout;
+        this.testServerTimeout = timeout;
+        this.latencyBasedRoutingTimeout = 3000;
```

Comment:
This should not be hardcoded, but should be a configurable parameter, since this is a library. It can have default value of 3000.